test/unit/org/apache/cassandra/io/sstable/LegacySSTableTest.java

```diff
@@ -612,7 +612,7 @@ private static void loadLegacyTable(String tablePattern, String legacyVersion) t
      * during development. I.e. remove the {@code @Ignore} annotation temporarily.
      * </p>
      */
-    @Ignore
+//    @Ignore
```

Comment:
I believe we want this `@Ignore` back, once the serialization samples have been generated

Reply:
ah yes, indeed